build.sbt

```diff
@@ -22,24 +36,25 @@ lazy val releaseSettings = Seq(
   organizationName := "National Archives",
   scmInfo := Some(
     ScmInfo(
-      url("https://github.com/nationalarchives/dr2-preservica-client"),
-      "git@github.com:nationalarchives/dr2-preservica-client.git"
+      url("https://github.com/nationalarchives/dr2-disaster-recovery"),
+      "git@github.com:nationalarchives/dr2-disaster-recovery.git"
     )
   ),
   developers := List(
     Developer(
       id = "tna-digital-archiving-jenkins",
       name = "TNA Digital Archiving",
       email = "digitalpreservation@nationalarchives.gov.uk",
-      url = url("https://github.com/nationalarchives/dr2-preservica-client")
+      url = url("https://github.com/nationalarchives/dr2-disaster-recovery")
     )
   ),
   description := "A client to communicate with the Preservica API",
```

Comment:
This description is for the Preservica client

Reply:
Good spot, I've changed it.